config/jobs-cip.yaml

```diff
@@ -125,119 +62,235 @@ jobs:
       - 'next'
```

Comment:
Does this section mean that we only build for the `arm64`, `next` and `sashal-next` trees?

Should we be building for the `cip` tree instead?

Reply:
was added only for testing, should be removed now

Reply:
no means that we also build for arm64 next and sashal-next. this section is inherited section